lib/SIL/SILPrinter.cpp

```diff
@@ -1050,6 +1050,11 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
     *this << ARI->getType();
   }
 
+  void visitReallocRefInst(ReallocRefInst *ARI) {
```

Comment:
I noticed in a bunch of places you copied the variable names from the alloc ref inst case. Can you change them to more sensible names for realloc ref inst (for instance I would name this RRI for **R**ealloc**R**ef**I**nst

Reply:
:)